integration-tests/jest/jest-utils.js

```diff
@@ -69,30 +75,38 @@ const flakeShaker = ({ fn, name }) => {
       }
     })
 
+    process.stdout.write(`Running tests: 0%`)
     await cluster.task(async ({ page, data }) => {
       const { taskFn, taskParams, runId } = data
       try {
         const result = await taskFn({ context, page, ...taskParams })
-        rawConsole.log(`Test number ${runId} passed`)
         return result
       } catch (e) {
         await page.screenshot({ path: `${screenshotDir}/${runId}.jpg`, fullPage: true })
-        rawConsole.log(`Test number ${runId} failed: ${e}`)
         return e
+      } finally {
+        process.stdout.clearLine()
```

Comment:
This looks better, but when running this against a test like `link-to-new-workspace` I still see the 'Running tests' line disappear.

I have an idea to correct this, let's pair tomorrow.